navBar.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css" type="text/css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.1/css/bulma.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  <script defer src="https://use.fontawesome.com/releases/v5.1.0/js/all.js"></script>
+  <script>
+    $(document).ready(function() {
+      $("#navBar").load("navBar.html");
+    });
+  </script>
+  <script src="js/scripts.js"></script>
+</head>
+
+<body>
+  <div class="navigationBar">
+    <!-- <nav>
+      <ul>
+        <li><a href="index.html">CityView</a></li>
+        <div class="dropdown">
+          <button onclick="myFunction()" class="dropbtn">Dropdown</button>
+          <div id="myDropdown" class="dropdown-content">
+            <a href="html/category1.html">category1</a>
+            <a href="html/livability.html">Livability</a>
+            <a href="html/events.html">Events</a>
+          </div>
+        </div>
+        <li><a href="about.html"></a>About</li>
+      </ul>
+    </nav> -->
+    <!-- Navbar -->
+    <nav class="navbar is-spaced has-shadow is-mobile">
```

Comment:
According to the docs at https://bulma.io/documentation/components/dropdown/
it looks like you need to contain the dropdown in a 'dropdown-trigger' with 'is-hoverable' so it knows to keep the downdown active while the mouse is hovering over it